### DISCUSSION TITLE
Fix website on Google Play

### DISCUSSION THREAD

**Discussion by maximepvrt** [community/community#62594] [Mobile]
### Select Topic Area

Bug

### Body

Can you change the website on Google Play App page [https://play.google.com/store/apps/details?id=com.github.android](https://play.google.com/store/apps/details?id=com.github.android) form `https://github.com/community/community/discussions/categories/mobile` to `https://github.com/orgs/community/discussions/categories/mobile`

**Comment by mxie** [comment]
Great catch @maximepvrt! We've made the update.
